Gemfile.lock

```diff
@@ -120,8 +120,8 @@ GEM
     bullet (6.0.2)
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
-    byebug (11.0.1)
-    capistrano (3.11.2)
+    byebug (11.1.1)
```

Comment:
### [byebug](https://github.com/deivid-rodriguez/byebug)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 11.0.1 → 11.1.1

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/byebug)_]





<details>
<summary>Commits</summary>

A change of **98** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/byebug/compare/v11.0.1...v11.1.1).

These are the first 10 commits:
- (d16d829) [Drop ruby 2.3 support (#554)](https://github.com/deivid-rodriguez/byebug/commit/d16d8291fe2b8b5125f31b8521af8a9ec2c88496)
- (bf41a63) [Bump rubocop from 0.64.0 to 0.68.0 (#555)](https://github.com/deivid-rodriguez/byebug/commit/bf41a63858a648baa7fb621600d6451786d1572a)
- (1ccbf0e) [Fix intermittent test hanging (#561)](https://github.com/deivid-rodriguez/byebug/commit/1ccbf0e3632db3879e5dd7633d0fc2a5c734a4c7)
- (493586a) [Bump rubies to latest patch levels](https://github.com/deivid-rodriguez/byebug/commit/493586a4f6c4ce0a86588d70d8e216f6219771bb)
- (ef6adef) [Ignore ruby-head from main rake docker tasks](https://github.com/deivid-rodriguez/byebug/commit/ef6adef82bb621f8611dd1f4ea33d2b87cf76765)
- (3769a2f) [Simplify bundler binstub](https://github.com/deivid-rodriguez/byebug/commit/3769a2f75b92f1812addd37cb05f8d491b3b29ad)
- (f9c70fb) [Bump development dependencies](https://github.com/deivid-rodriguez/byebug/commit/f9c70fb0011f50dca9b6b2190db265b86a009585)
- (59dd8aa) [Use rubygems 3.0.3 for testing](https://github.com/deivid-rodriguez/byebug/commit/59dd8aac65702a89df4d0772c3589d659bebe046)
- (9a9e023) [Bump alpine in test docker image](https://github.com/deivid-rodriguez/byebug/commit/9a9e0232c2c1290bfa20f76a0fef63dbde980531)
- (961620d) [Merge pull request #560 from deivid-rodriguez/bump_rubies](https://github.com/deivid-rodriguez/byebug/commit/961620d0e239ee07e5f2776b04fb90703f8db2ee)


</details>